### DISCUSSION TITLE
New data table look with filters / labels in the Assist devices

### DISCUSSION THREAD

**Discussion by VivantSenior** [home-assistant/frontend#21360] [Accessibility] [answered]
Hi,

I'm missing new data table look in `/config/voice-assistants/assist/devices` (like in https://my.home-assistant.io/redirect/devices/). As far as I remember @frenck stated that the new look and features should be implemented everywhere in HA (for reference: [2024.4](https://www.home-assistant.io/blog/2024/04/03/release-20244/#upgraded-tables) and [2024.5](https://www.home-assistant.io/blog/2024/05/01/release-20245/#more-features-for-our-data-tables)).

Best,
Viv

**Comment by frenck** [comment]
Well spotted 👀

**Comment by VivantSenior** [comment]
@frenck, do you need something more from my side? Bug report, feature request?

**Reply by frenck** [reply, marked as answer]
No nothing needed.
